src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 100;
+    private static final int FIGURES_COUNT = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();

```

Comment:
The 'random' field should be a constant. In Java, instances of 'Random' are typically declared as 'static final'.